.github/workflows/merge-queue.yml

```
@@ -231,6 +231,7 @@ jobs:
           ssh-key: ${{ secrets.HELM_VALUES_SSH_KEY }}
           app-id: ${{ secrets.HELM_VALUES_APP_ID }}
           branch: ${{ needs.prepare.outputs.MQ_HELM_VALUES_BRANCH }}
+          commit-msg: ${{ steps.manifest.MQ_COMMIT_MSG }}
```

actionlint (1.7.4)

property "mq_commit_msg" is not defined in object type {conclusion: string; outcome: string; outputs: {string => string}}

(expression)